package.json

```diff
@@ -20,6 +20,7 @@
     "url": "https://github.com/google/web-stories-wp/issues"
   },
   "dependencies": {
+    "@welldone-software/why-did-you-render": "^4.1.3",
```

Comment:
We already have one of these.

Reply:
True, this won't be in any final changes.

Reply:
This library is a little different from useWhyDidYouRender, in that it monkey patches all of the pure components automatically, and shows context hook changes.